### PR TITLE
Updated espocrm for 16.1

### LINKED ISSUE #1363: EspoCRM - include "WebSocket" in next EspoCRM appliance release
A EspoCRM user recently noted that he was having troubles setting up EspoCRM's [WebSocket](https://github.com/espocrm/documentation/blob/master/docs/administration/websocket.md).

It looks fairly straight forward and well documented, so we should definitely include that by default in our appliance.

The only other additional requirement would be installation of the `php-zmq` package.

[updated websocket documentation url]

### PR DESCRIPTION
**Pull request by OnGle**
Closes https://github.com/turnkeylinux/tracker/issues/1363

### CODE REVIEW

**Comment by OnGle** [comment]
Oops forgot the espocrm-websockets unit file (now included)

File: changelog [hunk 1]
```diff
@@ -1,3 +1,12 @@
+turnkey-espocrm-16.1 (1) turnkey; urgency=low
+
+  * Updated EspoCRM to v6.1.2.
+
```
Comment (JedMeister) [inline changelog:4]: Would be good to also note the inclusion of the websocket (and that it closes issue XXX) plus also note the new requirement for a domain.

**Review by JedMeister — CHANGES_REQUESTED** [review]
Could you please just add a note re the closed issue to the changelog? Thanks in anticipation... :smile:

File: changelog (hunk 1 above)
Comment (OnGle) [reply changelog:4]: No worries, please see the 2 additional commits

**Pull request merged**